### DISCUSSION TITLE
Google Drawing

### DISCUSSION THREAD

**Discussion by agusramdan** [gityopie/odoo-addons#208] [Ideas]
HI

Saya melihat feature google darawing untuk membuat geo fence. Saya pikir akan lebih baik lagi bila data di simpan dalam format geojson sehingga bisa menampung beberapa poligon dalam 1 field.

Salam
AMR

**Comment by gityopie** [comment]
Halo @agusramdan 

Terima kasih sarannya, bakalan sy cek
